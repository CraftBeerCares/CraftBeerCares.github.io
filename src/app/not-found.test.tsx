import { render } from "@testing-library/react";
import NotFound from "./not-found";

it("renders Not Found", () => {
  const { container } = render(<NotFound />);
  expect(container).toMatchSnapshot();
});
