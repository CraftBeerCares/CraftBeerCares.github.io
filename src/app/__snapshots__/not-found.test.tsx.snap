// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Not Found 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <button
        class="burgerMenuButton"
      >
        Menu
      </button>
      <h1
        class="className title"
      >
        Craft Beer Cares
      </h1>
      <nav
        class="topNav className"
      >
        <a
          class="navLink"
          href="/"
        >
          Home
        </a>
        <a
          class="navLink"
          href="/about"
        >
          About
        </a>
        <a
          class="navLink"
          href="/join_us"
        >
          Join Us
        </a>
        <a
          class="navLink"
          href="/events"
        >
          Events
        </a>
        <a
          class="navLink"
          href="/shop"
        >
          Shop
        </a>
        <a
          class="navLink"
          href="/contact"
        >
          Contact
        </a>
      </nav>
    </header>
    <main
      class="mainContent"
    >
      <h1>
        Page Not Found
      </h1>
    </main>
    <footer
      class="footer"
    >
      <div>
        Craft Beer Cares
      </div>
      <nav />
      <div>
        <img
          alt="Facebook"
          data-nimg="1"
          decoding="async"
          height="30"
          loading="lazy"
          src="/assets/facebook.svg"
          style="color: transparent;"
          width="30"
        />
        <img
          alt="Instagram"
          data-nimg="1"
          decoding="async"
          height="30"
          loading="lazy"
          src="/assets/instagram.svg"
          style="color: transparent;"
          width="30"
        />
        <img
          alt="Twitter"
          data-nimg="1"
          decoding="async"
          height="30"
          loading="lazy"
          src="/assets/twitter.svg"
          style="color: transparent;"
          width="30"
        />
      </div>
      <div>
        Craft Beer Cares - Drinking beer for good Stuff since 2017
      </div>
    </footer>
  </div>
</div>
`;
